compiler.rst

```diff
@@ -10,16 +10,16 @@ Abstract
 Historically (through 2.4), compilation from source code to bytecode
 involved two steps:
 
-1. Parse the source code into a parse tree (Parser/pgen.c)
-2. Emit bytecode based on the parse tree (Python/compile.c)
+1. Parse the source code into a parse tree (:file:`Parser/pgen.c`)
+2. Emit bytecode based on the parse tree (:file:`Python/compile.c`)
```

Comment:
This comment is meant to be part of review below.  I clicked wrong button when submitting it.